portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,12 +26,28 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String quote="Hello Pranjali!";
-   
-    response.setContentType("text/html;");
-    response.getWriter().println(quote);
+    List<String> quotes = new ArrayList<>();
+    quotes.add(
+        "And I knew exactly what to do. But in a much more real sense, I had no idea what to do.- Michael G. Scott");
+    quotes.add(
+        "It is a capital mistake to theorize before one has data. Insensibly one begins to twist facts to suit theories, instead of theories to suit facts.- Sherlock Holmes");
+    quotes.add(
+        "When you are backed against the wall, break the goddamn thing down.- Harvey Specter");
+    quotes.add(
+        "Never forget what you are. The rest of the world will not. Wear it like armor, and it can never be used to hurt you.- Tyrion Lannister");
+    List<String> quotesList = new ArrayList<String>(quotes);
+    response.setContentType("application/json;");
```

Comment:
You don't need to make a copy of the quotes. You can pass 'quotes' to convertToJson.
response.getWriter().println(convertTijson(quotes));